classes/main.rb

```diff
@@ -0,0 +1,112 @@
+require 'json'
+require 'date'
+require_relative 'item'
+require_relative 'game'
+require_relative 'author'
+require_relative 'book'
+require_relative 'label'
+require_relative 'genre'
+require_relative 'music_album'
+require_relative 'create_models'
+require_relative 'loader'
+require_relative 'list_all'
+
+class Main
+  attr_accessor :authors, :items, :genres, :labels, :music_albums
+
+  def initialize
+    @authors = []
+    @items = []
+    @music_albums = []
+    @genres = []
+    @labels = []
+
+    @create_models = CreateModels.new(self)
+    @list_all = ListAll.new(self)
+    @actions = actions
+
+    load_records
+    run
+  end
+
+  private
+
+  def run
+    print_options
+    option = gets.chomp.to_i
+    @actions[option - 1].call
+    run unless option == 15
+  end
+
+  def actions
+    [
+      -> { @create_models.create_item }, -> { move_to_archive },
+      -> { @list_all.list_all_books }, -> { @list_all.list_all_labels },
+      -> { @list_all.list_all_games }, -> { @list_all.list_all_authors },
+      -> { @list_all.list_all_genres }, -> { @list_all.list_all_albums },
+      -> { @create_models.add_book }, -> { @create_models.add_label },
+      -> { @create_models.add_genre }, -> { @create_models.add_music_album },
+      -> { @create_models.add_author }, -> { @create_models.add_game },
+      -> { quit }, -> { puts 'Invalid option. Please choose a valid option.' }
+    ]
+  end
+
+  def print_options
+    puts 'Options:'
+    puts '1. Create Item'
+    puts '2. Move Item to Archive'
+    puts '3. List all books'
+    puts '4. List all labels'
+    puts '5. List all games'
+    puts '6. List all authors'
+    puts '7. List all genres'
+    puts '8. List all albums'
+    puts '9. Add a book'
+    puts '10. Add a label'
+    puts '11. Add a genre'
+    puts '12. Add a music album'
+    puts '13. Add an author'
+    puts '14. Add a game'
+    puts '15. Quit'
+
+    print 'Choose an option: '
+  end
```

Comment:
- [ ] Great job on the project requirements but you should not provide options in the menu to directly add `label`, `genre`, and `author` as requested in the project requirements.  You should ask for these inputs from users while creating  `book`, `music album`, and `games`. You should also not provide the option to add `item`, the `Item` class is the parent class for other classes so it can be used for inheritance. Please make sure to fix this issue.